xgcm/grid.py

```diff
@@ -1080,13 +1080,20 @@ def __init__(
         face_connections : dict
             Grid topology
         coords : dict, optional
-            Explicit specification of axis coordinates, e.g
+            Specifies positions of dimension names along axes X, Y, Z, e.g
```

Comment:
@jdldeauna I would love to get some feedback on these new docstrings. Do you think these are sufficiently clear? Or do you have any suggestions how to improve these?

Reply:
Looks great! I just had a minor suggestion:
```python
Keys should be one of `X`, `Y`, or `Z` and map to a dictionary 
            mapping positions 
```
maybe
```python
Each key should be an axis name (e.g., `X`, `Y`, or `Z`) and map to a dictionary which maps positions 
```

Reply:
Thanks a lot, that is even more clear. I have integrated it and also added a few more details in the following sentence. What do you think @jdldeauna?